test/e2e/restart.go

```diff
@@ -48,9 +48,7 @@ const (
 	restartPodReadyAgainTimeout = 5 * time.Minute
 )
 
-// TODO(ihmccreery): This is skipped because it was previously in
-// REBOOT_SKIP_TESTS, dates back before version control (#10078)
-var _ = Describe("Restart [Skipped]", func() {
+var _ = Describe("Restart [Disruptive]", func() {
```

Comment:
this does mean that this test will now run by default on no-focus runs - is that what we want?